google-cloud-trace/lib/google/cloud/trace/faraday_middleware.rb

```diff
@@ -84,6 +109,14 @@ def add_response_labels span, env
         def set_label labels, key, value
           labels[key] = value if value.is_a? ::String
         end
+
+        ##
+        # @private Set X-Cloud-Trace-Context for request header
+        def set_trace_context_header env
+          if trace_ctx = Google::Cloud::Trace.get&.trace_context
```

Comment:
Please use `Stackdriver::Core::TraceContext.get`. i.e.
```ruby
if trace_ctx = Stackdriver::Core::TraceContext.get
```

Reply:
Got it, updated